### PR TITLE
Documentation week 10

### PR DESCRIPTION
**Pull request by ajjajjajjajj**
Add documentation for lead status feature and started some docs for transactions.

### CODE REVIEW

**Comment by WesleyBLDC** [comment]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -154,6 +154,28 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Lead Status feature
+
+The Lead Status feature aims to provide information about the contact based on when the status was last set.
+It is represented by the `LeadStatus status` attribute in a `Person`, which contains a `LeadStatusName` and 
+`LocalDateTime` timestamp.
```
Comment (evanpy) [inline docs/DeveloperGuide.md:161]: `
### Lead Status feature

#### About
`


I've added a sub heading "about" for my feature description!
Perhaps can here too for consistency.

File: docs/diagrams/StatusSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+!include style.puml
+
+
+participant ":StatusCommand" as StatusCommand LOGIC_COLOR
+participant ":Model" as Model LOGIC_COLOR
```
Comment (evanpy) [inline docs/diagrams/StatusSequenceDiagram.puml:7]: Might be a good idea to separate colors for visual aid!

**Review by evanpy — COMMENTED** [review]
Few possible tweaks. Otherwise, LGTM 👍🏻

**Pull request merged**